I18N.js

```diff
@@ -23,10 +23,10 @@ const Defaults = {
 const I18N = {
 
 	locales: [],
-	collator: new Intl.Collator(this.locales, Defaults.collatorOptions),
-	dateFormatter: new Intl.DateTimeFormat(this.locales, Defaults.dateFormatterOptions),
-	dateTimeFormatter: new Intl.DateTimeFormat(this.locales, Defaults.dateTimeFormatterOptions),
-	numberFormatter: new Intl.NumberFormat(this.locales, Defaults.numberFormatterOptions),
+	collator: new Intl.Collator([], Defaults.collatorOptions),
+	dateFormatter: new Intl.DateTimeFormat([], Defaults.dateFormatterOptions),
+	dateTimeFormatter: new Intl.DateTimeFormat([], Defaults.dateTimeFormatterOptions),
+	numberFormatter: new Intl.NumberFormat([], Defaults.numberFormatterOptions),
```

Comment:
fyi: `I18N.locales` should also work. `this` is indeed undefined in this context.

Reply:
I was unsure if one can really access that property already while still being _in the very definition itself_?

Reply:
I think it does, but the way it is now is the very secure way of doing it ;-)